### ISSUE TITLE
storageFilePath / storageRegistry : support of absolute path names

### ISSUE THREAD

**Issue by GitMensch** [kdcro101/vscode-favorites#64]
**Is your feature request related to a problem? Please describe.**
Especially when sharing favorites' settings between workspaces or between multiple users hard-coded paths are a pain. Using paths outside of the workspace can be quite tricky, support for absolute paths could help there.

**Describe the solution you'd like**
With a (mapped drive) `P:`: entries in `storageRegistry` that start with `P:\\` should work - similar `/mapped/` under Linux environments.

This is related but different to #63, I guess both may be implemented together.

**Comment by halo** [comment]
Is this a duplicate of https://github.com/kdcro101/vscode-favorites/issues/38 ?

**Comment by GitMensch** [comment]
> Is this a duplicate of #38 ?

No, #38 is about the files contained in the registry (defaulting to absolute names), this issue and the related one are about the `storageRegistry` entry itself.
